tests/test_api.py

```diff
@@ -180,6 +180,34 @@ def test_userinfo():
     assert result == expected
 
 
+@httpretty.activate
+def test_nodeinfo():
+    httpretty.register_uri(httpretty.GET, "http://dummyhost:80/system/cluster/nodes",
```

Comment:
this is maybe something that I should have done - we're not being very DRY where the `dummyhost` is concerned 😅

perhaps we could refactor its name, and the port, into variables instead of repeating the constant string / int?

if you feel like taking care of this, I would appreciate it, if not, just ignore this ^^

Reply:
I'll see what I can do :)
I'm no expert by any means :)